Fix twists

rr- opened the pull request:
#### Checklist

- [x] I have read the [coding conventions](https://github.com/LostArtefacts/TRX/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change
- [x] I have added a readme entry about my new feature or OG bug fix, or it is a different change

#### Description

This resolves #4078 and should be considered a draft as TR1 needs similar treatment.
It's not ideal, there's one frame where Lara is rotated wrong. But I can't do much about it.

Cross ref https://github.com/LostArtefacts/TRXInjectionTool/pull/136

rr- committed:
matrix: use quaternion rotations

In theory this should get rid of problems around 180° rotations, in
practice it does not.

rr- committed:
ui: enable in photo mode

walkawayy reviewed (approved):
Looks very nice to me! Great job.

lahm86 committed:
lara: fix twist animations

Restores missing frames in the jump back roll start and running jump
roll start animations, and additionally updates the turn 180 command
to include Lara's move angle.

lahm86 reviewed (approved):
LGTM. Thanks!

walkawayy commented:
Twist looks super clean now! 🎉

aredfan reviewed (approved):
Third time's the charm. 😌 

LGTM. 👍

Richard-L commented:
Been busy all day. Thanks a lot for prioritising this. Behaviour looks consistent again.
